src/cluster_legacy.h

```diff
@@ -4,7 +4,7 @@
 #define CLUSTER_PORT_INCR 10000 /* Cluster port = baseport + PORT_INCR */
 
 /* The following defines are amount of time, sometimes expressed as
- * multiplicators of the node timeout value (when ending with MULT). */
+ * multipliers of the node timeout value (when ending with MULT). */
```

Comment:
I could be wrong. It's possible this is intended to be some term-of-art, but I'm hoping the simpler word achieves the author's intent.